Core/src/IfcGeometryConverter/GeometrySettings.h

```diff
@@ -126,11 +126,20 @@ namespace OpenInfraPlatform
 
 				/*! returns the precision of the model
 				
+				
 				*/
-				double getPrecision() const
+				static double getPrecision()
+				{
+					return carve::EPSILON; //TODO remove constant and replace with content from IFC (i.e. introduce member, getter / setter)
+				}
+				/* ADDED static value to function, so it can be called whith other functions inside the class (e.c areEqual)
+				but it calls other problem which a type "const" in there.=>remove const

```

Comment:
I seriously doubt that the function cannot be const.

I don't object `static`, although it is not entirely clear to me, as to why?

Reply:
This was discussed during the call, please revert these changes and rather introduce overloads to the `areEqual` function:

    bool areEqual( const double, const double);
    bool areEqual( const double, const double, const double);

where the first calls the second with the default precision `getPrecision()`.

Reply:
Resolved: b2f7040